_software/SLiM.md

```diff
@@ -7,6 +7,21 @@ docs_url: https://messerlab.org/slim/
 category: simulate
 permalink: /SLiM
 code_snippet: |2
-  ##TODO##
+  initialize() {
+      initializeTreeSeq();
+      initializeMutationRate(1e-8);
+      initializeMutationType("m1", 0.5, "e", 0.001);
+      initializeGenomicElementType("g1", m1, 1.0);
+      initializeGenomicElement(g1, 0, 999999);
+      initializeRecombinationRate(1e-8);
+  }
+  1 { sim.addSubpop("p1", 500); }
+  2000 late() { sim.treeSeqOutput("out.trees"); }
 ---
-Here is some content that is on the full page about SLiM
\ No newline at end of file
+SLiM is an evolutionary simulation framework that combines a powerful engine
+for population genetic simulations with the capability of modeling arbitrarily
+complex evolutionary scenarios. The underlying individual-based simulation
+engine is highly optimized to enable modeling of entire chromosomes in
+large populations. There is also a graphical user interface on macOS and Linux
```

Comment:
```suggestion
large populations. There is also a graphical user interface
```
People run it on Windows also...

Reply:
True - this is simply copied straight off the main SLiM website - you might want to point out to Ben that he can change that there too?